pandas/tests/test_graphics.py

```diff
@@ -74,13 +74,23 @@ def setUp(self):
                                       'weight': random.normal(161, 32, size=n),
                                       'category': random.randint(4, size=n)})
 
-        if str(mpl.__version__) >= LooseVersion('1.4'):
+        mpl_version = str(mpl.__version__)
+        if mpl_version >= LooseVersion('1.4') or mpl_version[0] == '0':
             self.bp_n_objects = 7
         else:
             self.bp_n_objects = 8
+        # Thier dev version is messed up, 0+unknown
+        if mpl_version[0] == '0' or mpl_version >= LooseVersion('1.5'):
```

Comment:
prob best to put these in some functions (either in ``util.testing`` or top-level here)